src/priority_q.py

```diff
@@ -0,0 +1,32 @@
+from bin_heap import BinHeap
+
+
+class PQueue(object):
+    """Priority Queue object."""
+
+    def __init__(self):
+        """Initialize priority queue object."""
+        self._bh = BinHeap()
+        self._heap = self._bh._heap
+        self._count = 0
+
+    def insert(self, priority, data):
+        """Create a node and insert into p-queue."""
```

Comment:
That being said, your docstring could be more specific. What are the specifications for the two arguments? Should the priority argument be a certain data type?